project/src/com/company/Main.java

```diff
@@ -0,0 +1,15 @@
+package com.company;
+
+public class Main {
+
+
+    public static void main(String[] args) {
```

Comment:
![image](https://user-images.githubusercontent.com/67580205/161416169-968bd348-6c9e-44b5-80d6-7114ab88552e.png)
se comporta ciudat daca nu exista deja un cont. ori ascunde optiunea aceea, daca nu exista un cont, ori afiseaza un mesaj corespunzator
![image](https://user-images.githubusercontent.com/67580205/161416251-9a007c4b-71f1-49fb-a1d5-a530a38b9e44.png)
nu te astepta ca utilizatorii sa foloseasca corect aplicatia, multi sunt complet paralel cu tehnologia si o sa bage tot felul de aberatii

Reply:
Update: Momentan am rezolvat doar bug-urile in care incercam sa folosim o comanda ce are nevoie de date, atunci cand nu exista date introduse in aplicatie. Urmeaza sa rezolv si problemele cand se tasteaza un string atunci cand se cere un numar si viceversa.

Reply:
Am rezolvat toate problemele de input(cel putin cele pe care le-am gasit). 
Singura problema de input care ramane ar fi atunci cand se tasteaza o data cu format invalid(care sa nu fie ZZ LUNA AA).
Aceasta problema apare doar atunci cand incercam sa sortam in functie de data. In rest, se poate primi ca input(care in sine e o mare problema).
Solutia ar fi sa ma folosesc fie de Regular Expressions ca sa detectez daca formatul tasatat este valid fie(intr-un caz mai complicat) sa imi implementez propria clasa Date care sa se ocupe cu input-ul pentru date calendaristice(pentru format-ul si context-ul acestei aplicatii).

Commit:
https://github.com/AmarKayed/Advanced-OOP-Project/commit/13492375569df799cfa5e9ea9ba50ff41cfe356f